xblockmufi/xblockmufi.py

```diff
@@ -129,3 +195,54 @@ def build_fragment(self,
         if fragment_js:
             fragment.initialize_js(fragment_js)
         return fragment
+
+    @XBlock.json_handler
+    def student_submit(self, submissions, suffix=''):
```

Comment:
For consistency, it's probably best to keep the param named `data` instead of switching to `submissions` here.